fem/bilininteg_diffusion.cpp

```diff
@@ -645,9 +1069,9 @@ void PADiffusionApply3D(const int NE,
    auto G = Reshape(g.Read(), Q1D, D1D);
    auto Bt = Reshape(bt.Read(), D1D, Q1D);
    auto Gt = Reshape(gt.Read(), D1D, Q1D);
-   auto op = Reshape(_op.Read(), Q1D*Q1D*Q1D, 6, NE);
-   auto x = Reshape(_x.Read(), D1D, D1D, D1D, NE);
-   auto y = Reshape(_y.ReadWrite(), D1D, D1D, D1D, NE);
+   auto D = Reshape(d_.Read(), Q1D*Q1D*Q1D, 6, NE);
```

Comment:
You can use `auto D = Reshape(d_.Read(), Q1D, Q1D, Q1D, 6, NE);` to avoid computing `const int q = qx + (qy + qz * Q1D) * Q1D;` (this applies in many different kernels)